src/main/proto/tech/figure/loan/v1beta1/loan_documents.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package tech.figure.loan.v1beta1;
+
+option java_multiple_files = true;
+
+import "tech/figure/util/v1beta1/document.proto";
+
+/*
+List of loan documents for a single loan
+ */
+message LoanDocuments {
+  repeated tech.figure.util.v1beta1.DocumentMetadata document = 1; // Individual document metadata w/ pointer to actual document
```

Comment:
Is this proto defined just so it can be a separate fact? otherwise I think the idea was it could be just one of the `kv`s